### ISSUE TITLE
TypeScript Declarations

### ISSUE THREAD

**Issue by styfle** [sindresorhus/is-port-reachable#5]
Interested in PR with `index.d.ts`?

I'm currently using this:

```ts
declare module 'is-port-reachable' {
  export interface IsPortReachableOptions {
    timeout?: number | undefined;
    host?: string;
  }
  export default function(
    port: number | undefined,
    options?: IsPortReachableOptions
  ): Promise<boolean>;
}

```

**Comment by sindresorhus** [comment]
Sure, if you're willing to do the effort to follow [this styleguide](https://github.com/sindresorhus/typescript-definition-style-guide).

Some examples:
- https://github.com/sindresorhus/semver-regex/commit/039944bb5b3876913171b895488874b5c200e458
- https://github.com/sindresorhus/global-dirs/commit/0aae4d64887f77af94c4fb7ca12aa9f93c0a8291
- https://github.com/sindresorhus/md5-hex/commit/bbbfeff840f26973fbffc16d4863ae6d120079f3

**Comment by mikicho** [comment]
#8

**Comment by mikicho** [comment]
closed by #8
